modules/images/webp-uploads/load.php

```diff
@@ -573,3 +575,43 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Returns an array of image sizes that can have alternative mime types. By default, only core sizes
+ * are allowed and will be returned.
+ *
+ * If a custom size wants to have additional mime types support, they will need to have
```

Comment:
Replace with
"Developers can control the generation of additional mime images for all sizes using this filter."

We can add the bit about adding via `add_image_size` later when we support that.

Reply:
Updated the text but decided not to add information about `add_image_size` yet.